homeassistant/components/microbees/api.py

```diff
@@ -0,0 +1,28 @@
+"""API for microBees bound to Home Assistant OAuth."""
+
+from homeassistant.const import CONF_ACCESS_TOKEN
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_entry_oauth2_flow
+
+
+class ConfigEntryAuth:
```

Comment:
This isn't used anywhere. The idea with this class is that it should inherit the library client class and help the library to refresh the access token since Home Assistant is in control of it instead of the library.

https://developers.home-assistant.io/docs/api_lib_auth#oauth2